pkg/apis/servicecatalog/v1alpha1/conversion.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+// ServicePlanFieldLabelConversionFunc does not convert anything, just returns
```

Comment:
Can we get a comment as to why these are needed?  Any why only these two classes and not the rest of the svc-cat model?

Reply:
done